.github/workflows/kill_idle_cons.yml

```diff
@@ -0,0 +1,40 @@
+# Kill idle connections older than 10 minutes
+name: Kill idle connections
```

Comment:
## The action
Runs every 3 hours. Chose that number because today the problem stopped and started again within 4 hours.